### ISSUE TITLE
Distinction between route and track needed?

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#8830]
Right now we have two menu entries in map menu:
- Individual track
- Individual route

IMHO most users are unaware of this difference as both come within a GPX (most of the time associated to a cache as an aid to perform the cache).
Shouldn't we make it "one thing" from users perspective (meaning UI)?

**Comment by moving-bits** [comment]
Actually we have three things currently:
- GPX files in "track" format
- GPX files in "route" format
- manually created routes

The first two are already combined in the UI, although handled separately internally (calculating the route point connections using brouter).

The third one is currently called "individual route", but should be called "manual route" instead.

**Comment by Lineflyer** [comment]
OK, understood so far.
But I don't understand why the menu "Individual route" -> "Load indivudual route" brings me to a screen where I can select a GPX and get an error in return.

Why isn't is compatible to any other GPX?

Sorry for dumb asking, but should saving an manually created route which was export, not result in a track (or route) being able to be loaded by the other menu option?

**Comment by fm-sys** [comment]
Maby we could handle GPX files in "route" format like manual created routes and show their way points on livemap, so that the route could also be edited.

**Comment by moving-bits** [comment]
> OK, understood so far.
> But I don't understand why the menu "Individual route" -> "Load indivudual route" brings me to a screen where I can select a GPX and get an error in return.

> Why isn't is compatible to any other GPX?

The "individual route file" is a standard GPX file in route format and can be read as such.
But it does not work the other way around currently, as the "name" field is expected to be a valid geocode.

> Sorry for dumb asking, but should saving an manually created route which was export, not result in a track (or route) being able to be loaded by the other menu option?

An exported "individual route" can be read using "load individual route" (then still being able to be edited).

Since #8810 (extension to "load individual track" to be able to read GPX route files as well) it should be readable using "load individual track" as well, but have not tested yet.

**Comment by moving-bits** [comment]
I do have the following items on my to do list for routes & tracks:

- rename "Individual route" to "Manual route"
- rename "Individual track" to "GPX track/route"
- enable "manual route" to read regular "GPX route files" to be able to extend/change routes loaded from arbitrary GPX route files

How about the names - would that help to clarify? Any other suggestions on how to name (and differentiate) the two items?

**Comment by fm-sys** [comment]
Yes, that would help to clarify which function is for what use case

**Comment by fm-sys** [comment]
But when we rename "individual track" to "GPX track /route" I don't really see any pros of renaming "individual route". I whould leave it like it is, also because I personally like the name...
